SwaprHQ/stackly-ui
# [SWA-191][FEAT] - Add Mainnet support for stackly

## Fixes: [STK-191](https://linear.app/swaprhq/issue/STK-191/add-mainnet-deployment-for-stackly)

### Description
* Overwrite selected StartTime for the Stackbox, if the date is in the past

### How to test the changes
1) Pull this branch
2) Run the project locally
3) Go to the Stackly homepage, connect your wallet
4) Top right network selector should allow you to connect to Ethereum network
5) Try creating a stack
